rosetta-source/src/main/resources/cdm-sample-files/fpml-5-13/products/commodity-derivatives/expectations.json

```diff
@@ -5,11 +5,11 @@
+{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex37-gold-forward-offered-rate.xml","excludedPaths":-1,"externalPaths":97,"outstandingMappings":1,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex4-electricity-swap-hourly-off-peak.xml","excludedPaths":-1,"externalPaths":62,"outstandingMappings":6,"validationFailures":3,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Swap_FixedFloat","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex41-oil-asian-barrier-option-strip.xml","excludedPaths":-1,"externalPaths":83,"outstandingMappings":30,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex41-oil-asian-barrier-option-strip.xml","excludedPaths":-1,"externalPaths":83,"outstandingMappings":28,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex46-simple-financial-put-option.xml","excludedPaths":-1,"externalPaths":62,"outstandingMappings":8,"validationFailures":7,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex48-gold-forward-offered-rate-ois.xml","excludedPaths":-1,"externalPaths":96,"outstandingMappings":1,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis_OIS","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex48-gold-forward-offered-rate-ois.xml","excludedPaths":-1,"externalPaths":96,"outstandingMappings":1,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis_OIS","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
Validation errors due to use of commodity floating rate index which does not match any FloatingRateIndexEnum values